mscore/prefsdialog.ui

```diff
@@ -1160,6 +1166,16 @@
            </widget>
           </item>
           <item row="1" column="0">
+           <widget class="QCheckBox" name="enableAutoSplit">
+            <property name="accessibleName">
+             <string>Auto-split note duration into tied notes on important measure subdivisions</string>
```

Comment:
Maybe "Regroup rhythms while entering notes," with the more complicated text available in a tooltip?